src/set_relation/set_relation.cc

```diff
@@ -2938,9 +2938,9 @@ void VisitorBoundDomainRange::preVisitUFCallTerm(UFCallTerm * t){
     // Determine what the output arity of this particular UF call is
     // taking into consideration that it could be indexed.
     unsigned int out_arity = range->arity();
-    if (uf_call->isIndexed()) {
-        out_arity = 1; // only one element is being accessed
-    } 
+//    if (uf_call->isIndexed()) {
```

Comment:
I understand what this means semantically. However, it has no positive use in the implementation. at least non that I could find. And this is actually causing one of the test cases to fail (NormalizationTest.buildingNormalize).